### DISCUSSION TITLE
静态博客写作新体验-小书匠

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#290] [Announcements]
https://bbing.com.cn/202206/staticblog_xiaoshujiang/

**Comment by caibingcheng** [comment]
TODO: 图片有点问题

**Comment by caibingcheng** [comment]
突然觉得好亏...用邀请码注册的话，相当于40块钱可以买两年了...亏了亏了呀:< 我是新用户，但是没有邀请码，也不好意思去问行不行了QAQ

> 拥有邀请码的用户，在注册为小书匠用户后的一个月内升级为会员，系统将赠送一年(365天)的会员时长。

校园邮箱的用户相当于40块钱三年？

> 只要使用了带有 edu 域名的邮箱用户，在完成邮箱验证后进行付款，成为小书匠会员后，自动奖励两年( 365*2 天)的会员时长。

[http://soft.xiaoshujiang.com/price/](http://soft.xiaoshujiang.com/price/)

**Comment by penndu** [comment]
看起来像专为HEXO这种静态博客定制的！

**Comment by W4J1e** [comment]
其实在线写作平台还有一些大佬在用的语雀之类，也有大佬为hexo这样的静态博客开发过“后台”。我从未尝试，原因是我的博客和github其实没多大关系，每次都只能手动push到服务器再去刷新cdn缓存。

**Comment by caibingcheng** [comment]
> @W4J1e
> 其实在线写作平台还有一些大佬在用的语雀之类，也有大佬为hexo这样的静态博客开发过“后台”。我从未尝试，原因是我的博客和github其实没多大关系，每次都只能手动push到服务器再去刷新cdn缓存。

语雀方案挺好的, 不过图片外链问题得折腾一下

**Comment by penndu** [comment]
我用的vscode，图片手动外链。

**Comment by caibingcheng** [comment]
> @penndu
> 我用的vscode，图片手动外链。

在这之前我也是用的vscode，搭配github codesapce就可以在线编辑，不需要本地环境

**Comment by penndu** [comment]
> @caibingcheng
> > @penndu
> > 我用的vscode，图片手动外链。
> 
> 在这之前我也是用的vscode，搭配github codesapce就可以在线编辑，不需要本地环境

之前一直用github.dev，本地用notepad++，后来出现辱华事件之后，就换成vscode。
